.github/workflows/regression-tests.yml

```diff
@@ -19,13 +19,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04]
-        compiler: [g++-10, clang++-15]
```

Comment:
I think this is where GCC 10 support is removed? I'd like to keep it please (see longer comment in main thread).

Reply:
This is just changing the defaults. GCC 10 is kept on Ubuntu 20.04 in new lines 29 - 30. It might be possible to install that version on the newer Ubuntu, but I don't this there will be much difference in the test execution. I can check this, though.

Reply:
D'oh! Right you are, sorry. I didn't look at the rest of the changes carefully enough.